.coderabbit.yaml

```
@@ -0,0 +1,22 @@
+reviews:
+  # Enable reviews for cleanup branches specifically
+  include_patterns:
+    - "coderabbit-cleanup"
+    - "cleanup/*" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)